tests/framework/prototype.py

```diff
@@ -506,7 +506,7 @@ def tearDown(self) -> None:
 
     rs = RuleSet([HeadPodNameRule(), EasyJobRule(), HeadSvcRule()])
     mut = Mutator(base_cr, patch_list)
-    images = ['rayproject/ray:2.0.0', 'kuberay/operator:nightly', 'kuberay/apiserver:nightly']
```

Comment:
Do we also like to test 2.0.0 in parallel?

Reply:
Eventually.
This test is a prototype.